modular/gater/metadata_handler.go

```diff
@@ -277,14 +276,21 @@ func (g *GateModular) listObjectsByBucketNameHandler(w http.ResponseWriter, r *h
 		ContinuationToken:     continuationToken,
 	}
 
-	respBytes, err = xml.Marshal(grpcResponse)
+	if format == "json" {
```

Comment:
Supporting JSON format in all metadata API will be implemented soon. Here just implementing the json response for one API so that later others can follow the pattern.